raigad/src/main/java/com/netflix/raigad/resources/ElasticsearchHealthCheck.java

```diff
@@ -0,0 +1,35 @@
+package com.netflix.raigad.resources;
+
+/**
+ * Created by alfasi on 4/9/15.
+ */
+import com.netflix.raigad.utils.ElasticsearchProcessMonitor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/v1/healthcheck")
+@Produces(MediaType.APPLICATION_JSON)
+public class ElasticsearchHealthCheck {
+
+    private static final long serialVersionUID = 1L;
+    private static final Logger log = LoggerFactory.getLogger(ElasticsearchHealthCheck.class);
+    private static final String REST_SUCCESS = "[\"ok\"]";
+
+    @GET
+    @Path("/")
```

Comment:
Please rename the path to something else (or may be v1/healthcheck), because in future if we add new method to this class and want to define new REST endpoint, it's better to have different endpoints for each method rather than one method having root path and others having extensions. 

eg. @Path("/esrunning") or something better if you can think of anything.